android/ReactAndroid/src/main/java/com/facebook/react/modules/network/OkHttpClientProvider.java

```diff
@@ -49,7 +49,7 @@ public static void replaceOkHttpClient(OkHttpClient client) {
 
     public static OkHttpClient createClient() {
         try {
-            return (OkHttpClient) Class.forName("host.exp.exponent.ReactNativeStaticHelpers").getMethod("getOkHttpClient", Class.class).invoke(null, OkHttpClientProvider.class);
+            return (OkHttpClient) Class.forName("host.exp.exponent.ReactNativeStaticHelpers").getMethod("getOkHttpUnprefixedClient", Class.class).invoke(null, OkHttpClientProvider.class);
```

Comment:
```suggestion
            return (OkHttpClient) Class.forName("host.exp.exponent.ReactNativeStaticHelpers").getMethod("getOkHttpClient", Class.class).invoke(null, OkHttpClientProvider.class);
```

Let's add name modifications to these extra functions and treat unprefixed client as the normal one